### PR TITLE
[G2M] Carousel

### PR DESCRIPTION
**Pull request by DoParkEQ**

### CODE REVIEW

File: src/imageSlider/carousel.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  return (
+    <div>
+      <Slider {...settings} ref={getRef}>
+        {content}
+      </Slider>
+    </div>
+  );
+};
+
+Carousel.PropTypes = {
```
Comment (woozyking) [inline src/imageSlider/carousel.js:30]: the reason why linter complains about no propTypes definition is that this needs to be `Carousel.propTypes`

Comment (DoParkEQ) [reply src/imageSlider/carousel.js:30]: 🤦

File: src/imageSlider/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  );
+};
+
+ImageSlider.propTypes = {
+  topSectionContent: PropTypes.object,
+  carouselContent: PropTypes.object,
+}
+
+ImageSlider.defaultProps = {
+  topSectionContent: {
```
Comment (DoParkEQ) [inline src/imageSlider/index.js:25]: Both `topSectionContent` and `carouselContent` are being passed to `<Carousel/>` via `<ImageSlider/>`. In this case, do those props still need to have default props both at parent and child level?

Comment (DoParkEQ) [reply src/imageSlider/index.js:25]: @woozyking

Comment (woozyking) [reply src/imageSlider/index.js:25]: no. in this case you can take a look at https://github.com/EQWorks/react-labs/blob/master/src/table/table-toolbar/index.js#L69 and see how the parent component `<TableToolbar />` stacks in the prop-types definitions of its children components `<Download />` and `<Toggle />`.

so usually it's better to define the prop-types at children's levels and reuse them to compose into parent levels, when necessary.

File: src/css/carousel.css [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.slick-slide {
+	padding-right: 16px;
+	box-sizing: border-box;
+ }
+
```
Comment (woozyking) [inline src/css/carousel.css:5]: let's keep using css-in-js

Comment (DoParkEQ) [reply src/css/carousel.css:5]: I wasn't sure how to append style attribute to '.slick-slide' through JSS as `<Slider/>` as it is using an external library. Would you be able to show me how?

**Pull request merged**